tool/site/client/src/pages/loadPage.tsx

```diff
@@ -122,10 +122,6 @@ const LoadPage: React.FC = () => {
                     value: "secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request",
                     text: "Secondary Care - Repeat Dispensing (nominated)"
                   },
-                  {
-                    value: "secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner",
-                    text: "Homecare - Acute (nominated)"
-                  },
```

Comment:
Couldn't really see why we've deleted this one

Reply:
our first of type users will not be using homecare and we're not confident it's a representative example so dropping until we're asked to make it available and had feedback on what's wrong with it